### DISCUSSION TITLE
모바일에서 GetComponent<T> 고쳐 졌나요?

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#745] [5. Q&A]
월드 1.3.0 입니다 
모바일에서 GetComponent<T> 안되던거 고쳐졌나요?

이거는되는데 
![image](https://user-images.githubusercontent.com/50188795/168195063-deebd4de-258e-4559-8fc5-2ebe95252eaa.png)
![image](https://user-images.githubusercontent.com/50188795/168195079-777c9590-b655-4f88-8aa8-a7e7c352fad0.png)

이거는 왜 안되는건가요? 
![image](https://user-images.githubusercontent.com/50188795/168195159-7215f535-d7bb-4826-b1ab-d0828a0cbdba.png)
![image](https://user-images.githubusercontent.com/50188795/168195210-bf4a83d4-f0a5-4d93-94bb-f422a38c9c8d.png)

요부분은 안되나바요 
```typescript
var controller2: PlayerController = target.GetComponent(typeof (PlayerController)) as PlayerController;
console.log(`controller2 --------------------> ${controller2}`);
var controller3: PlayerController = target.GetComponent("PlayerController") as PlayerController;
console.log(`controller3 --------------------> ${controller3}`);
```

![image](https://user-images.githubusercontent.com/50188795/168195981-f004ad6a-785e-4cb4-a2a8-16162956c249.png)

https://docs.unity3d.com/kr/530/ScriptReference/GameObject.GetComponent.html

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 
수정 진행 중에 있으며 수정 완료될 경우 패치 노트에 공개 예정입니다.
다만 적용 시기는 현재로서 공유드리기 어려운 점 양해 부탁 드립니다.

https://github.com/naverz/zepeto-studio-kor/discussions/528 이슈 참고 부탁 드립니다.
